src/components/tabs.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
```

Comment:
I think this component is too generic. You want to render just several buttons (one is active, other are grey), so maybe it's just better to use div and some buttons instead of creating abstracted version of it that will be used only in this place. 

BTW: There is already `Tabs` component:

![image](https://github.com/user-attachments/assets/45201354-e272-4d74-8e1f-9fe6c9d781d3)